cmd/secret/cmd/root.go

```diff
@@ -0,0 +1,83 @@
+// Package cmd provides functions set and get with cobra library
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-itools-internship/go-secret/pkg/crypto"
+	"github.com/go-itools-internship/go-secret/pkg/io/storage"
+	"github.com/go-itools-internship/go-secret/pkg/provider"
+	"github.com/spf13/cobra"
+)
+
+type root struct {
+	key       *string
+	cipherKey *string
+	value     *string
+	path      *string
+	cmd       *cobra.Command
+}
+
+func (r *root) Execute(ctx context.Context) error {
+	return r.cmd.ExecuteContext(ctx)
+}
+
+func New(version string) *root {
+	var secret = &cobra.Command{
+		Use:     "secret",
+		Short:   "Root includes cli commands",
+		Long:    "Create CLI to set and get secrets via the command line",
+		Version: version,
+	}
+	v := secret.PersistentFlags().StringP("value", "v", "", "value to be encrypted")
+	k := secret.PersistentFlags().StringP("key", "k", "", "key to encrypt value")
```

Comment:
Is this the key which is used for encrypting value or key that will be associated with value?